addons/mail/static/src/discuss/core/common/thread_model_patch.js

```diff
@@ -255,7 +254,14 @@ const threadPatch = {
             return this.correspondent.name;
         }
         if (this.channel_type === "group" && !this.name) {
-            return formatList(this.channel_member_ids.map((channelMember) => channelMember.name));
+            let name = this.channel_member_ids
+                .slice(0, 5)
+                .map((channelMember) => channelMember.name)
+                .join(", ");
+            if (this.channel_member_ids.length > 5) {
+                name += "...";
+            }
+            return name;
```

Comment:
There is no gain in this PR if we still have a compute in JS based on members.
Can't this be removed completely?

Also we should be removing groups from `channels_with_all_members = self.filtered(lambda channel: channel.channel_type != "channel")`, this would be the whole point of this PR not having to returns the members.